.github/workflows/build_whl_and_publish.yaml

```diff
@@ -0,0 +1,40 @@
+name: Build nightly wheels and publish to PyTorch Index
+
+on:
+  push:
```

Comment:
```suggestion
  pull_request:
  push:
```
@joecummings to test this workflow you can also enable it on pull request

Reply:
Yep, did this on past commits :)